tests/vm_watch_test.go

```diff
@@ -213,6 +214,77 @@ var _ = Describe("[rfe_id:3423][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		Expect(err).ToNot(HaveOccurred())
 		Expect(titles).To(Equal([]string{"NAME", "AGE", "STATUS", "VOLUME"}),
 			"Output should have the proper columns")
+
+		vmStatus, err := readNewStatus(stdout, titles, readTimeout)
```

Comment:
@zcahana Could you PTAL at `readNewStatus` and see if it's possible to refactor it in a non recursive manner? if it's a big change then let's do it in a separate PR.

Reply:
Doesn't look like a big change. Changing...